doc/sphinx/parameters/Formulation.md

```diff
@@ -41,7 +41,7 @@
   \item `custom': A custom selection of `Mass conservation' and `Temperature equation'.
 \end{enumerate}
 
-\note{Warning: The `custom' option is implemented for advanced users that want full control over the equations solved. It is possible to choose inconsistent formulations and no error checking is performed on the consistency of the resulting equations.}
+\note{Warning: The 'custom' option is implemented for advanced users that want full control over the equations solved. It is possible to choose inconsistent formulations and no error checking is performed on the consistency of the resulting equations.}
```

Comment:
That's the wrong place to fix these things -- this file is autogenerated from the source files. Whenever we update the `parameters.json` file, this change will be overwritten.

The solution is to replace these ticks by proper `&lsquo;` and `&rsquo;`.

Reply:
@sac-bsa Might be able to help you with that in the script that created these files. It should be a relatively straightforward regex replace thing.

Reply:
Oh, on second thought, this isn't generated from parameters.jason, is it?

Reply:
I know its something that will have to be addressed in a more formal capacity in another pull request (Timo brought it up in another discussion) but for the purposes of this pull request I changed that one instance so that everything works correctly in the test build.